frontend/src/result-list.js

```diff
@@ -374,7 +374,7 @@ export class ResultList extends React.Component {
   }
 
   getRuns() {
-    fetch(buildUrl(Settings.serverUrl + '/run', {pageSize: 1000}))
+    fetch(buildUrl(Settings.serverUrl + '/run', {pageSize: 20, estimate: true}))
```

Comment:
So, this fetches the list of runs so that the dropdown on the results list page can show and filter runs. I can't remember if we do a server-side lookup for the list, but limiting this to 20 might be a problem.

Reply:
Oh right, I think we should modify this dropdown so it can accept run_id's that aren't "pre-fetched". 

Furthermore, this dropdown is from before the runs table right? I don't think we should make an API call for 1000 runs when it is easier and more user friendly to use the runs table.

Reply:
I reverted this change (but kept the `estimate`), we can improve this API call, potentially with on-the-fly filtering in a future PR.